src/plugins/oops-utils.c

```diff
@@ -305,7 +305,7 @@ char *abrt_oops_string_filter_regex(void)
 
     load_abrt_plugin_conf_file("oops.conf", settings);
 
-    int only_fatal_mce = 1;
+    int only_fatal_mce = 0;
```

Comment:
*Just a note: This has no effect on the function.*

There is one issue with `OnlyFatalMCE` that requires a change in libreport.
```
abrt-dump-journal-oops[70081]: abrt-dump-journal-oops: Option 'OnlyFatalMCE' is not configured
```

See:
https://github.com/abrt/libreport/blob/460a9c40afda1c413042eb70c5b3dae2135ea521/src/lib/abrt_types.c#L80

Reply:
Thanks, I'll check these. There seem to be some more, possibly problematic calls to the `try_get_map_string_item_as_*` family.